flink-python/setup.py

```diff
@@ -308,10 +308,10 @@ def extracted_output_files(base_dir, file_path, output_directory):
         author='Apache Software Foundation',
         author_email='dev@flink.apache.org',
         python_requires='>=3.6',
-        install_requires=['py4j==0.10.9.3', 'python-dateutil==2.8.0', 'apache-beam==2.27.0',
+        install_requires=['py4j==0.10.9.3', 'python-dateutil==2.8.0', 'apache-beam==2.36.0',
                           'cloudpickle==1.2.2', 'avro-python3>=1.8.1,!=1.9.2,<1.10.0',
-                          'pandas>=1.0,<1.2.0', 'pyarrow>=0.15.1,<3.0.0',
-                          'pytz>=2018.3', 'numpy>=1.14.3,<1.20', 'fastavro>=0.21.4,<0.24',
+                          'pandas>=1.3.0', 'pyarrow>=5.0.0',
+                          'pytz>=2018.3', 'numpy>=1.20.3', 'fastavro>=0.21.4,<0.24',
```

Comment:
@deadwind4 
Can we upgrade numpy to >=1.21.0 ?
With 1.20.3 there are still problems with instalation on M1

Reply:
OK, This Beam PR is also created by me :)

Reply:
I know. Thanks.